docs/development/code_quality.rst

```diff
@@ -24,6 +24,8 @@ Black
 
 A better method is to run Black automatically - first `integrate it within the code editor <https://black.readthedocs.io/en/stable/editor_integration.html>`_ you use and then enable the "format on save" or "format on type" option in your editor settings.
 
+If you've installed pre-commit hooks (`pre-commit install`) Black will run before each commit.
```

Comment:
```suggestion
Alternatively, Black will run automatically before each commit if you've installed pre-commit hooks (as mentioned in git worklfow <add link to L154 there, using sphinx ref>)
```

Reply:
Thanks, I've changed it and added the ref.